build/install_xla_in_source_tree.sh

```diff
@@ -52,8 +52,6 @@ fi
 
 # Copy the XLA dependencies into jax/lib, fixing up some imports to point to the
 # new location.
-cp -f "$(rlocation __main__/jaxlib/lapack.so)" "${TARGET}/jaxlib"
```

Comment:
I think you didn't mean to drop the `lapack` line here?

Reply:
Thanks for the extra attention here. Sorry for the number of comments and attention you had to spend on this.